games/c.yaml

```
@@ -496,6 +496,22 @@
   type: remake
   updated: 2017-02-08
 
+- name: Civ2-clone
+  originals:
+  - Civilization II
+  type: remake
+  repo: 'https://github.com/axx0/Civ2-clone'
+  development: active
+  status: unplayable
+  content: commercial
+  lang:
+  - C#
+  framework:
+  - .NET
+  license:
+  - GPL3
+  updated: '2021-03-11'
```

just a silly question, isn't it better to put today's date?

```suggestion
  updated: '2021-03-21'
```